test/performance/common/setup-clusterloader2.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+# Copyright 2024 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -x # echo so that users can understand what is happening
+set -e # exit on error
+
+env="k8s"
+exp_type="s" # accepted values: a) l = long-running test; b) s = short-running test
+
+while [ $# != 0 ]; do
+    case "$1" in
+        (-h|--help) echo "$0 usage: (--env ( kind | ocp ) | --exp ( s | l )*"
+                    exit;;
+        (--env)
+          if (( $# > 1 )); then
+            env="$2"
+            shift
+          else
+            echo "Missing env value" >&2
+            exit 1;
+          fi;;
+        (--exp)
+          if (( $# > 1 )); then
+            exp_type="$2"
+            shift
+          else
+            echo "Missing exp value" >&2
+            exit 1;
+          fi;;
+    esac
+    shift
+done
+
+:
+: -------------------------------------------------------------------------
+: "Configuring clusterloader2 to generate KS performance workloads"
+
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)
+cp $SCRIPT_DIR/cleanup.sh $CL2_DIR/
```

Comment:
Suggestion: since this script simply _assumes_ that `$CL2_DIR` is defined, perhaps it should check and fail with an explanation if not.

Reply:
Fixed - added a check and explanation of how to set `$CL2_DIR`